desktop/src/main/java/bisq/desktop/components/TxIdTextField.java

```diff
@@ -173,7 +175,8 @@ private void openBlockExplorer(String txId) {
             BlockChainExplorer blockChainExplorer = isBsq ?
                     preferences.getBsqBlockChainExplorer() :
                     preferences.getBlockChainExplorer();
-            GUIUtil.openWebPage(blockChainExplorer.txUrl + txId, false);
+            String url = this.isAddress ? blockChainExplorer.addressUrl : blockChainExplorer.txUrl;
+            GUIUtil.openWebPage(url + txId, false);
```

Comment:
I don't think `TxIdTextField` should open an address URL. I think `AddressTextField` should do that just based on the name (I don't really know the code base yet)

Reply:
I agree - unfortunately  `AddressTextField` doesn't open web browser with blockchain explorer - it's purpose is to open wallet internally.